src/app/data/index.ts

```diff
@@ -0,0 +1,84 @@
+import { AppData, Game, Mod } from '~/models';
```

Comment:
This file looks like it was accidentally added as a duplicate

Reply:
Yes, duplicate